config/examples/simple.yaml

```diff
@@ -0,0 +1,8 @@
+# exclude s3 object that match `object.1*` but include anything else that match `object.*`
+s3:
+  include:
+    names_regex:
+    - object.*
```

Comment:
According to [the README](https://github.com/gruntwork-io/cloud-nuke/blob/master/README.md?plain=1#L638), the `s3` names_regex will filter on the bucket name, and not the object key. Are you sure this will work?

[This part of the README](https://github.com/gruntwork-io/cloud-nuke/blob/master/README.md?plain=1#L500) also says
> The intention is to delete all the **s3 buckets** that match the include rules but not the exclude rules

Reply:
I don't know what happened, but it looks like we had some regression here 🤷